services/frontend-service/src/ui/components/ProductVersion/ProductVersion.tsx

```diff
@@ -181,7 +186,12 @@ export const ProductVersion: React.FC = () => {
                     addAction({
                         action: {
                             $case: 'releaseTrain',
-                            releaseTrain: { target: env, commitHash: selectedTag, team: '' },
+                            releaseTrain: {
+                                target: env,
+                                commitHash: selectedTag,
+                                team: '',
+                                targetType: ReleaseTrainRequest_TargetType.UNKNOWN,
```

Comment:
I think we're just one call of `splitCombinedGroupName` away, to know if it's an env or a group.
Either do that, or add it to the follow-up story.